jetson/percep/pcl.cpp

```diff
@@ -157,11 +157,11 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
     for (int i = 0; i < (int)cluster_indices.size(); ++i)
     {
         std::vector<int>* curr_cluster = &interest_points[i];
-
+        
         //Initialize interest points
         std::fill(curr_cluster->begin(), curr_cluster->end(), cluster_indices[i].indices[0]);
         
-        //Order of interest points: 0=Up Left 1=Up Right 2=Low Right 3=Low Left
+
```

Comment:
Has the order changed or is this just removing documentation?

Reply:
I don't believe it has, probably was an accident. Will recommit the commit